api/usersd/service.go

```diff
@@ -561,6 +567,65 @@ func (s *Service) ArchiveRetrievalLogs(req *pb.ArchiveRetrievalLogsRequest, serv
 	return nil
 }
 
+func (s *Service) ListFilRewards(ctx context.Context, req *pb.ListFilRewardsRequest) (*pb.ListFilRewardsResponse, error) {
+	account, _ := mdb.AccountFromContext(ctx)
+	org := account.Org
+	user := account.User
+
+	if org == nil {
+		return nil, status.Error(codes.Unauthenticated, "no org provided")
+	}
+
+	if user != nil && user.Type != mdb.Dev {
+		return nil, status.Error(codes.Unauthenticated, "provided user must be a dev")
+	}
+
+	opts := []filrewards.ListOption{filrewards.WithOrgKeyFilter(org.Key.String())}
```

Comment:
Example of inferring/validating org and user information from the auth context and setting those values to send to the filrewardsd service.